qa-core/src/jest/jestSetup.ts

```diff
@@ -1,4 +1,5 @@
 import { logging } from "@budibase/backend-core"
 logging.LOG_CONTEXT = false
 
+jest.retryTimes(2)
```

Comment:
The only thing I'm uncertain of is the idempotency of the test run. Given they usually setup some data, set some global state, I'm not sure where the retry will/will not help - but worth a shot